scripts/automation/tests/verify_packages.py

```diff
@@ -75,6 +75,9 @@ def verify_packages():
     for name, path in all_modules:
         build_package(path, built_packages_dir)
 
+    # TODO Remove this when package released and merging to master
+    exec_command('python -m pip install git+https://github.com/Azure/azure-sdk-for-python.git@azurestack#egg=azure-mgmt-storage&subdirectory=azure-mgmt-storage')
```

Comment:
This will be removed before merging to master but needed currently as SDK not released.